mlf_core/create/templates/mlflow/mlflow_pytorch/{{ cookiecutter.project_slug }}/README.rst

```diff
@@ -3,16 +3,16 @@
 {{ cookiecutter.project_name }}
 {% for _ in cookiecutter.project_name %}={% endfor %}
 
-.. image:: https://github.com/{{ cookiecutter.github_username }}/{{ cookiecutter.project_slug }}/workflows/Train%20{{ cookiecutter.project_slug }}%20using%20CPU/badge.svg
-        :target: https://github.com/{{ cookiecutter.github_username }}/{{ cookiecutter.project_slug }}/workflows/Train%20{{ cookiecutter.project_slug }}%20using%20CPU/badge.svg
+.. image:: https://github.com/{{ cookiecutter.github_username }}/{{ cookiecutter.project_slug }}/workflows/Train%20{{ cookiecutter.project_slug }}%20using%20CPU/badge.svg?branch=main
```

Comment:
I used main branch here, cause this is the default for new projects. We should state this somewhere or add a default branch variable for cookiecutter?